lib/jq/rhmi-usage-common.jq

```diff
@@ -0,0 +1,129 @@
+module {
+  "name": "rhmi-usage-common",
+  "description": "shared functions used by rhmi-usage-report-* scripts",
+  "homepage": "https://github.com/integr8ly/rhmi-utils#readme",
+  "license": "Apache",
+  "author": "Jesse Sarnovsky",
+  "version": "1.0.0",
+  "jq": "1.6",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/integr8ly/rhmi-utils.git"
+  }
+};
+
+import "i8-helpers" as i8;
+
+def getNamespaceClasses:
+  [
+    .namespaces[] | {
+      ns: .metadata.name,
+      labels: .metadata.labels
+    } | {
+      ns,
+      nsClass: (
+        if .labels.rhmi=="true" or .labels.integreatly=="true" then
+          "rhmi"
+        elif .ns=="redhat-rhmi-operator" then
+          "rhmi"
+        elif .ns | test("^(default|kube-.+)$") then
+          #"kube"
+          "openshift"
```

Comment:
Should these be classified as OpenShift or user?

Reply:
I have classifications for both namespaces and pods, specifically because of the ones where user and rhmi provided system pods co-exist.  i.e. Both rhmi/system pods and user created qdrouter pods end up in the enmasse ns.  For these mixed ones, the nsClass is still rhmi, but pods within it can be either rhmi or user.

Reply:
Thanks @jessesarn